management/organization.go

```diff
@@ -379,3 +379,32 @@ func (m *OrganizationManager) DeleteMemberRoles(ctx context.Context, id string,
 	err = m.management.Request(ctx, "DELETE", m.management.URI("organizations", id, "members", memberID, "roles"), &body, opts...)
 	return
 }
+
+// ClientGrants retrieves the client grants assigned to an organization.
+//
+// See: https://auth0.com/docs/api/management/v2/organizations/get-client-grants
+func (m *OrganizationManager) ClientGrants(ctx context.Context, id string, opts ...RequestOption) (g *ClientGrantList, err error) {
+	err = m.management.Request(ctx, "GET", m.management.URI("organizations", id, "client-grants"), &g, applyListDefaults(opts))
+	return
+}
+
+// AssociateClientGrant assigns a client grant to an organization.
+//
+// See: https://auth0.com/docs/api/management/v2/organizations/post-client-grants
+func (m *OrganizationManager) AssociateClientGrant(ctx context.Context, id string, grantID string, opts ...RequestOption) (err error) {
+	body := struct {
```

Comment:
Going to reach out to see whether this body could change over time as that would impact whether we just accept a `grantID` (which would mean we can't really add to this in future) or the body (which would allow parameters to be added in future)

Reply:
Seems like this should be safe, also removed links for now as they'll 404. Will update in future